stingray/lightcurve.py

```diff
@@ -1400,8 +1400,8 @@ def from_lightkurve(lk):
         lk : `lightkurve.LightCurve`
             A lightkurve LightCurve object.
         """
-        return Lightcurve(time=lc.time, counts=lc.flux,
-                          err=lc.flux_err, input_counts=False)
+        return Lightcurve(time=lk.time, counts=lk.flux,
+                          err=lk.flux_err, input_counts=False, skip_checks=True,)
```

Comment:
Since `lightkurve` objects are something we don't deal with much, and they should be small-ish objects, probably better to default to `False` (still better: add a keyword to the method :) ).